test/samples-js-extra/AddProductExtraColumns/index.js

```diff
@@ -0,0 +1,18 @@
+module.exports = async function (context) {
```

Comment:
For these failure scenario functions please copy over the comments explaining what the function is verifying